client/landing/gutenboarding/index.tsx

```diff
@@ -202,6 +209,58 @@ function waitForCurrentUser(): Promise< User | undefined > {
 	} ).finally( unsubscribe );
 }
 
+async function checkAndRedirectIfSiteWasCreatedRecently() {
+	const shouldPathCauseRedirectForSelectedSite = () => {
+		return [ Step.CreateSite, Step.Plans, Step.Style ].some( ( step ) => {
+			if ( window.location.pathname.startsWith( `/${ GUTENBOARDING_BASE_NAME }/${ step }` ) ) {
+				return true;
+			}
+		} );
+	};
+
+	if ( shouldPathCauseRedirectForSelectedSite() ) {
+		const selectedSiteDetails = await waitForSelectedSite();
+
+		if ( selectedSiteDetails ) {
+			const createdAtString = selectedSiteDetails?.options?.created_at;
+			// "2020-05-11T01:08:15+00:00"
+
+			if ( createdAtString ) {
+				const createdAt = new Date( createdAtString );
+				const diff = Date.now() - createdAt.getTime();
+				const diffMinutes = diff / 1000 / 60;
+				if ( diffMinutes < 10 && diffMinutes >= 0 ) {
+					window.location.replace( `/block-editor/page/${ selectedSiteDetails.ID }/home` );
```

Comment:
Was the plan to redirect to customer home?

`window.location.replace( `/home/${ selectedSiteDetails.ID }` );`

Or is that for another PR?

Reply:
Yes, I guess we should do that when the user is coming back from the editor.